MinPlatformPkg/Test/Library/TestPointCheckLib/TraditionalMmTestPointCheckLib.c

```diff
@@ -288,20 +253,18 @@ TestPointSmmReadyToBootSmmPageProtection (
         );
     }
   }
+
   DEBUG ((DEBUG_INFO, "======== TestPointSmmReadyToBootSmmPageProtection - Exit\n"));
   return EFI_SUCCESS;
 }
 
 /**
   Dispatch function for a Software SMI handler.
-
   Caution: This function may receive untrusted input.
   Communicate buffer and buffer size are external input, so this function will do basic validation.
-
   @param CommBuffer      A pointer to a collection of data in memory that will
                          be conveyed from a non-SMM environment into an SMM environment.
   @param CommBufferSize  The size of the CommBuffer.
-
   @retval EFI_SUCCESS Command is handled successfully.
 **/
 EFI_STATUS
```

Comment:
Same for this function. The standalone mm implementation in StandaloneMmTestPointCheckLib and the traditional MM implementation seems functionally identical?

Reply:
Consolidated.